src/main/java/net/mcreator/ui/dialogs/UpdatePluginDialog.java

```diff
@@ -41,24 +41,37 @@ public static void showPluginUpdateDialogIfUpdatesExist(Window parent) {
 
 			pan.add("North", L10N.label("dialog.plugin_update_notify.message"));
 			pan.add("Center", new JScrollPane(PanelUtils.pullElementUp(plugins)));
+			pan.setBorder(BorderFactory.createEmptyBorder(10, 10, 10, 10));
 			pan.setPreferredSize(new Dimension(560, 250));
 
 			for (PluginUpdateInfo pluginUpdateInfo : PluginLoader.INSTANCE.getPluginUpdates()) {
-				JLabel label = L10N.label("dialog.plugin_update_notify.version_message",
+				StringBuilder usb = new StringBuilder(L10N.t("dialog.plugin_update_notify.version_message",
 						pluginUpdateInfo.plugin().getInfo().getName(), pluginUpdateInfo.plugin().getInfo().getVersion(),
-						pluginUpdateInfo.newVersion());
+						pluginUpdateInfo.newVersion()));
+				if (pluginUpdateInfo.recentChanges() != null) {
+					usb.append("<br>").append(L10N.t("dialog.plugin_update_notify.changes_message")).append("<ul>");
+					for (String change : pluginUpdateInfo.recentChanges())
+						usb.append("<li>").append(change).append("</li>");
+				}
 
+				JLabel label = new JLabel(usb.toString());
 				JButton update = L10N.button("dialog.plugin_update_notify.update");
 				update.addActionListener(e -> DesktopUtils.browseSafe(
 						MCreatorApplication.SERVER_DOMAIN + "/node/" + pluginUpdateInfo.plugin().getInfo()
 								.getPluginPageID()));
-
-				plugins.add(PanelUtils.westAndEastElement(label, update));
+				plugins.add(PanelUtils.westAndEastElement(label, PanelUtils.join(update)));
 			}
 
-			JOptionPane.showOptionDialog(parent, pan, L10N.t("dialog.plugin_update_notify.update_title"),
-					JOptionPane.OK_CANCEL_OPTION, JOptionPane.PLAIN_MESSAGE, null,
-					new Object[] { L10N.t("dialog.plugin_update_notify.close") }, "");
+			MCreatorDialog window = new MCreatorDialog(null, L10N.t("dialog.plugin_update_notify.update_title"));

```

Comment:
Dialog should have a parent

Reply:
It is shown over a splash screen which is not a dialog or frame but just window, and changing the `SplashScreen`'s parent class changes the way SS is displayed (adds unwanted white background).

Reply:
![image](https://user-images.githubusercontent.com/16374228/235086688-f3e364a7-e5ac-4cb9-b445-67da44b81bf5.png)

Reply:
Maybe, but going that way results into a _"Wrong parent"_ exception which I can not explain 🤷‍♂️

Reply:
This one: `java.lang.IllegalArgumentException: Wrong parent window`.

Reply:
I see, seems only JOptionPane can handle JWindow, but not regular dialogs and windows